rmf_demos/launch/hotel.launch.xml

```diff
@@ -19,7 +19,7 @@
           exec="experimental_lift_watchdog"
           name="experimental_lift_watchdog"
           output="both" />
-    <set_env name="EXPT_LIFT_WATCHDOG_SRV" value="experimental_lift_watchdog_service"/>
+    <set_env name="EXPT_LIFT_WATCHDOG_SRV" value="experimental_lift_watchdog"/>
```

Comment:
Silly question.. Where does this environment variable actually get used? I can't find any reference to it in `rmf_demos` or in `rmf_ros2`.

Reply:
This is used in the launch file  [here](https://github.com/open-rmf/rmf_demos/blob/6de2a7655f2deb2c62f0c7890fd7608a97cdae2e/rmf_demos/launch/hotel.launch.xml#L70). The reason of using ENV is because `launch.xml `'s `<arg>` element doesn't support the "if/unless"  condition.